packages/components/src/reaction_icon_bubble/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import styles from './index.module.scss';
+import ReactionIcon from '../reaction_icon';
+import { REACTION_SIZE_M, REACTION_SIZE_S } from '../constants';
+
+const getPaddingForSize = (size) => {
+  if (size <= REACTION_SIZE_S) return 4;
+  if (size <= REACTION_SIZE_M) return 8;
+  return 9;
+};
+
+const ReactionIconBubble = ({
+  className: passedClassName,
+  reaction,
+  size,
+  colorize,
+  filled = false,
+}) => {
+  const className = clsx(passedClassName, styles.root, styles[`is-${reaction}`], {
+    [styles.isFilled]: filled,
+    [styles.isColorized]: colorize,
+  });
+
+  return (
+    <span className={className} style={{ padding: getPaddingForSize(size) }}>
```

Comment:
could you do this with %?

Reply:
Unfortunately no, values are too different